### PR TITLE
Change 'n+1' to '1+n' for better definitional behaviour

### PR DESCRIPTION
**Pull request by felixwellen**

### CODE REVIEW

File: SyntheticGeometry/ProjectiveSpace.lagda.md [hunk 1]
```diff
@@ -58,15 +59,15 @@ open import SyntheticGeometry.SQC.Consequences k k-local k-sqc
...
   private
     module k = CommRingStr (snd k)
-    module 𝔸ⁿ⁺¹ = LeftModuleStr (snd (FinVecLeftModule (CommRing→Ring k) {n = n + 1}))
+    module 𝔸ⁿ⁺¹ = LeftModuleStr (snd (FinVecLeftModule (CommRing→Ring k) {n = 1 + n}))
   open k hiding (_+_)
   open 𝔸ⁿ⁺¹ hiding (_+_)
   open Units k
 
-  𝔸ⁿ⁺¹ = FinVec ⟨ k ⟩ (n + 1)
+  𝔸ⁿ⁺¹ = FinVec ⟨ k ⟩ (1 + n)
```
Comment (MatthiasHu) [inline SyntheticGeometry/ProjectiveSpace.lagda.md:67]: Do we really want to keep the name `𝔸ⁿ⁺¹` if we actually use `1 + n` now?

File: SyntheticGeometry/ProjectiveSpace.lagda.md [hunk 2]
```diff
@@ -58,15 +59,15 @@ open import SyntheticGeometry.SQC.Consequences k k-local k-sqc
 module _ (n : ℕ) where
   private
     module k = CommRingStr (snd k)
-    module 𝔸ⁿ⁺¹ = LeftModuleStr (snd (FinVecLeftModule (CommRing→Ring k) {n = n + 1}))
+    module 𝔸ⁿ⁺¹ = LeftModuleStr (snd (FinVecLeftModule (CommRing→Ring k) {n = 1 + n}))
```
Comment (MatthiasHu) [inline SyntheticGeometry/ProjectiveSpace.lagda.md:62]: (also here)

File: SyntheticGeometry/ProjectiveSpace/LineThroughPoints.lagda.md [hunk 3]
```diff
@@ -116,7 +116,7 @@ module _
 
   private
     module k = CommRingStr (snd k)
-    𝔸ⁿ⁺¹-as-module = FinVecLeftModule (CommRing→Ring k) {n = n ℕ.+ 1}
+    𝔸ⁿ⁺¹-as-module = FinVecLeftModule (CommRing→Ring k) {n = 1 ℕ.+ n}
```
Comment (MatthiasHu) [inline SyntheticGeometry/ProjectiveSpace/LineThroughPoints.lagda.md:119]: (and here)

File: SyntheticGeometry/ProjectiveSpace.lagda.md (hunk 1 above)
Comment (MatthiasHu) [reply SyntheticGeometry/ProjectiveSpace.lagda.md:67]: If we want to keep the name, we should add a comment about `n + 1` vs `1 + n` here.

Comment (felixwellen) [reply SyntheticGeometry/ProjectiveSpace.lagda.md:67]: I decided for a comment.

**Review by MatthiasHu — APPROVED** [review]
Yes, good, let's merge!

**Pull request merged**